ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/Runner.java

```diff
@@ -60,53 +61,55 @@
   private final HttpResponseFactory responseFactory = new HttpResponseFactory();
   private final Path dataDirectory;
 
-  private final Vertx vertx;
+  private Vertx vertx;
```

Comment:
you can PROBABLY final this, and 'new' it here.

Reply:
I really like that we are newing it in the start and closing it in the stop. To me that makes a lot of sense ...